docs/applications/containers/introduction-to-microservices-with-kubernetes-and-gitlab/index.md

```diff
@@ -0,0 +1,210 @@
+gitlab-migrations.1-7npbx                              0/1     PodInitializing   0          119s
+gitlab-minio-c64f9694b-pskvl                           1/1     Running           0          119s
+gitlab-minio-create-buckets.1-qjclq                    1/1     Running           0          119s
+gitlab-nginx-ingress-controller-5846ccbf4-5m9kx        1/1     Running           0          118s
+gitlab-nginx-ingress-controller-5846ccbf4-6pl5g        1/1     Running           0          118s
+gitlab-nginx-ingress-controller-5846ccbf4-7d4sv        1/1     Running           0          118s
+gitlab-nginx-ingress-default-backend-7f87d67c8-9b6x9   1/1     Running           0          118s
+gitlab-nginx-ingress-default-backend-7f87d67c8-c7vqd   1/1     Running           0          118s
+gitlab-postgresql-cb4c58788-jhjkz                      2/2     Running           0          118s
+gitlab-prometheus-server-66f6d79d66-rrdtp              0/2     PodInitializing   0          117s
+gitlab-redis-6cc5f876db-hd44h                          0/2     PodInitializing   0          117s
+gitlab-registry-7f95847fb4-2x5bp                       1/1     Running           0          117s
+gitlab-registry-7f95847fb4-7cpg6                       1/1     Running           0          117s
+gitlab-sidekiq-all-in-1-74655c6c75-vcwp6               0/1     Init:2/3          0          119s
+gitlab-task-runner-68775bf44d-q27lh                    1/1     Running           0          119s
+gitlab-unicorn-694748c5c9-mb8kf                        0/2     Init:2/3          0          103s
+gitlab-unicorn-694748c5c9-w47tl                        0/2     Init:2/3          0          119s
+{{< /output >}}
+
+    The Helm chart will create a [NodeBalancer](/docs/platform/nodebalancer/getting-started-with-nodebalancers/), which will be used to add an additional layer of high availability to our application.
+
+1. Domain propagation will take a few minutes to successfully resolve to your new IP address following this change, and the `gitlab-runner` microservice may crash since it relies on this resolution. You'll want to periodically check this service to make sure it hasn't failed. See the [Troubleshooting section](/docs/applications/containers/introduction-to-microservices-with-kubernetes-and-gitlab/#troubleshooting) for tips on how to resolve Pods in statuses other than `Running`.
+
+        kubectl get pods | grep gitlab-runner
+
+1. Navigate to the [NodeBalancer](https://cloud.linode.com/nodebalancers) section of the Cloud Manager, and find the IP address of this new resource. The label of the NodeBalancer will be a randomly generated 32 character string, with your worker nodes pre-configured as Backend Linodes.
+
+1. If you have a domain set-up to point to one of your worker nodes as instructed above, as your containers come up you should see A records made automatically for each of the subdomains. They should point to your NodeBalancer. If not, you can create them manually.
```

Comment:
As we discussed I decided to rearrange this and say that these should be created automatically, but if not, they can create them manually.